### PR TITLE
Removed duplicate cuisine translations.

### PR DESCRIPTION
**Pull request by deathbaba**
Fixes bug found when editing cuisines.

### CODE REVIEW

File: generator/osm2meta.cpp [hunk 1]
```diff
@@ -5,8 +5,59 @@
...
+{
+  unordered_set<T> seen;
+  auto const predicate = [&seen](T const & value)
+  {
+    if (seen.find(value) != seen.end())
+      return true;
+    seen.insert(value);
+    return false;
+  };
+  vec.erase(std::remove_if(vec.begin(), vec.end(), predicate), vec.end());
```
Comment (mgsergio) [inline generator/osm2meta.cpp]: sort + unique не подойдет?

File: generator/osm2meta.cpp [hunk 2]
```diff
@@ -5,8 +5,59 @@
...
+  string GetString()
+  {
+    if (m_values.empty())
+      return string();
+
+    RemoveDuplicatesAndKeepOrder(m_values);
+    string result;
+    for (auto const & str : m_values)
+      result += str + kOSMMultivalueDelimiter;
+    result.resize(result.size() - 1);
```
Comment (mgsergio) [inline generator/osm2meta.cpp]: Есть JoinStrings в base/string_utils.hpp

**Comment by deathbaba** [comment]
@mgsergio Fixed, merge please.

**Comment by mgsergio** [comment]
LGTM

**Pull request merged**